src/app/reservation/hotel-cards/hotel-cards.component.scss

```diff
@@ -1,24 +1,111 @@
 @import '../../../scss/grid.scss';
+@import '../../../scss/break-points.scss';
 
 
 .rooms {
   max-height: 200px;
   min-height: 200px;
+  // max-height: 100%;
+  // min-height: 100%;
   max-width: 670px;
   min-width: 670px;
   display: flex;
-  background: red;
+  background: whitesmoke;
   margin: 25px;
+  // box-shadow: 0px 1px 5px 0px rgba(153,147,153,1);
+  box-shadow: 0px 1px 5px 0px rgba(240,237,240,1);
+  @media screen and (max-width: $break) {
+    // max-height: 500px;
+    // min-height: 500px;
+    max-height: 100%;
+    min-height: 100%;
+    max-width: 350px;
+    min-width: 350px;
+    display: block;
+    margin-bottom: 30px;
+    padding-bottom: 5px;
+  }
 }
 
 .text-container {
   display: flex;
   flex-direction: column;
+  margin: 25px;
+  margin-top: 15px;
+  @media screen and (max-width: $break) {
```

Comment:
$break should indicate what breakpoint it is, like 'large', 'medium', 'small' or 'handheld'